documentation/book/con-maintenance-time-window-definition.adoc

```diff
@@ -0,0 +1,25 @@
+// Module included in the following assemblies:
+//
+// assembly-maintenance-time-windows.adoc
+
+[id='con-maintenance-time-window-definition-{context}']
+= Maintenance time window definition
+
+You configure maintenance time windows by entering an array of strings in the `Kafka.spec.maintenanceTimeWindows` property.
+Each string is a {CronExpression} interpreted as being in UTC (Universal coordinated time, which for practical purposes is the same as Greenwich Mean Time).
```

Comment:
Initial caps: UTC (Coordinated Universal Time). Do we need the part about GMT?

Reply:
Does everyone know what UTC is, these days?

Reply:
https://trends.google.com/trends/explore?cat=13&q=UTC,GMT would seem to suggest that both terms are used at about the same frequency overall, but with significant regional variation. I therefore think it's _probably_ worth keeping, but happy to remove it if people want.

Reply:
Nice metrics. Let's keep the reference to GMT. @ppatierno - please update the acronym to Coordinated Universal Time (UTC).

Reply:
I already fixed that, but github isn't showing you the most up to date version.